src/Components/Card.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace NotificationChannels\Facebook\Components;
+
+use NotificationChannels\Facebook\Exceptions\CouldNotCreateCard;
+use NotificationChannels\Facebook\Exceptions\CouldNotCreateMessage;
+use NotificationChannels\Facebook\Exceptions\CouldNotSendNotification;
+
+
+class Card implements \JsonSerializable
+{
+    /** @var string Card Title */
+    protected $title;
+
+    /** @var string Item Url */
+    protected $image_url;
+
+    /** @var string Image Url */
+    protected $item_url;
+
+    /** @var string Subtitle */
+    protected $subtitle;
+
+    /** @var array Call to Action Buttons */
+    public $buttons = [];
+
+    /**
+     * Create a Card
+     *
+     * @return static
+     */
+    public static function create($title = '')
+    {
+        return new static($title);
+    }
+
+    /**
+     * Create Card constructor
+     */
+    public function __construct($title = '')
+    {
+        $this->title($title);
+    }
+
+    /**
+     * Set Button Title.
+     *
+     * @param $title
+     * @throws CouldNotCreateCard
+     * @return $this
+     */
+    public function title($title)
+    {
+        if (mb_strlen($title) > 80) {
+            throw CouldNotCreateCard::titleLimitExceeded($this->title);
+        }
+        $this->title = $title;
+        return $this;
+    }
+
+    /**
+     * Add up to 3 call to action buttons.
+     *
+     * @param array $buttons
+     *
+     * @return $this
+     * @throws CouldNotSendNotification
+     */
+    public function buttons(array $buttons = [])
+    {
+        if (count($buttons) > 3) {
+            throw CouldNotCreateMessage::messageButtonsLimitExceeded();
+        }
+        $this->buttons = $buttons;
+        return $this;
+    }
+
```

Comment:
This `buttons` method seems to be duplicated in another file. Hence, It's better we move this to a trait instead. Please let me know if you can do it or I'll do it after merging.